lib/send_sms.rb

```diff
@@ -0,0 +1,15 @@
+require 'twilio-ruby'
+require 'dotenv/load'
+
+@account_sid = ENV["TWILIO_ACCOUNT_SID"]
+@auth_token = ENV["TWILIO_AUTH_TOKEN"]
+@client = Twilio::REST::Client.new @account_sid, @auth_token
+
+message = @client.messages.create(
+  body: "Thanks for your order! It will be delivered at approximately #{@time.strftime("%H:%M")}.",
+  to: ENV["MY_PHONE_NUMBER"],
+  from: ENV["TWILIO_MAGIC_NUMBER"])
+
+def delivery_time
+  @time = Time.new + 45 * 60
+end
```

Comment:
I'm guessing you used this as a note pad and its not part of the production code.

Reply:
Will eventually be used to call the sms function from its own class, but for now I've just added this straight to the order.rb file.